### ISSUE TITLE
Dynamic Trees compatibility

### ISSUE THREAD

**Issue by Mic1e** [Cristelknight/Expanded-Ecosphere#62]
Hi,
Any possibility to have compatibility with Dynamic Trees? Currently only WWOO biomes don't work with DT trees from what I've seen. Would love to use both mods together. (Using 1.19.2 at the moment).

**Comment by Doc-1** [comment]
I am looking into making a Dynamic Tree add-on for WWEE and came across this post. Is this enhancement currently being worked on? If not, I'd be happy to continue working on it. I would love to see these two mods together. Thanks!

**Comment by Cristelknight** [comment]
No. There's nothing being worked on or really planned. So would be cool if you do it. Maybe link the page here when you're done!
